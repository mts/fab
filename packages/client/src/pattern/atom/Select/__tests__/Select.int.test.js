import { dogSelectRender, catSelectRender } from './Select.int.render'

describe('<Select />', () => {
  describe('Snaphot', () => {
    test('must match dogSelectRender', () => {
      expect(global.renderToJSON(dogSelectRender)).toMatchSnapshot()
    })

    test('must match catSelectRender', () => {
      expect(global.renderToJSON(catSelectRender)).toMatchSnapshot()
    })
  })
})
