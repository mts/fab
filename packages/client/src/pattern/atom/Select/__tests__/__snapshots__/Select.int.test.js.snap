// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select /> Snaphot must match catSelectRender 1`] = `
<select
  className="select"
>
  <option
    value="Abyssinian"
  >
    Abyssinian
  </option>
  <option
    value="Aegean"
  >
    Aegean
  </option>
  <option
    value="American-Bobtail"
  >
    American Bobtail
  </option>
  <option
    value="American-Curl"
  >
    American Curl
  </option>
  <option
    value="American-Shorthair"
  >
    American Shorthair
  </option>
  <option
    value="American-Wirehair"
  >
    American Wirehair
  </option>
  <option
    value="Arabian-Mau"
  >
    Arabian Mau
  </option>
  <option
    value="Australian-Mist"
  >
    Australian Mist
  </option>
  <option
    value="Balinese"
  >
    Balinese
  </option>
  <option
    value="Bambino"
  >
    Bambino
  </option>
  <option
    value="Bengal"
  >
    Bengal
  </option>
  <option
    value="Birman"
  >
    Birman
  </option>
  <option
    value="Bombay"
  >
    Bombay
  </option>
  <option
    value="British-Longhair"
  >
    British Longhair
  </option>
  <option
    value="British-Shorthair"
  >
    British Shorthair
  </option>
  <option
    value="Burmese"
  >
    Burmese
  </option>
  <option
    value="Burmilla"
  >
    Burmilla
  </option>
  <option
    value="California-Spangled"
  >
    California Spangled
  </option>
  <option
    value="Chantilly-Tiffany"
  >
    Chantilly-Tiffany
  </option>
  <option
    value="Chartreux"
  >
    Chartreux
  </option>
  <option
    value="Chausie"
  >
    Chausie
  </option>
  <option
    value="Cheetoh"
  >
    Cheetoh
  </option>
  <option
    value="Colorpoint-Shorthair"
  >
    Colorpoint Shorthair
  </option>
  <option
    value="Cornish-Rex"
  >
    Cornish Rex
  </option>
  <option
    value="Cymric"
  >
    Cymric
  </option>
  <option
    value="Cyprus"
  >
    Cyprus
  </option>
  <option
    value="Devon-Rex"
  >
    Devon Rex
  </option>
  <option
    value="Donskoy"
  >
    Donskoy
  </option>
  <option
    value="Dragon-Li"
  >
    Dragon Li
  </option>
  <option
    value="Egyptian-Mau"
  >
    Egyptian Mau
  </option>
  <option
    value="European-Burmese"
  >
    European Burmese
  </option>
  <option
    value="Exotic-Shorthair"
  >
    Exotic Shorthair
  </option>
  <option
    value="Havana-Brown"
  >
    Havana Brown
  </option>
  <option
    value="Himalayan"
  >
    Himalayan
  </option>
  <option
    value="Japanese-Bobtail"
  >
    Japanese Bobtail
  </option>
  <option
    value="Javanese"
  >
    Javanese
  </option>
  <option
    value="Khao-Manee"
  >
    Khao Manee
  </option>
  <option
    value="Korat"
  >
    Korat
  </option>
  <option
    value="Kurilian"
  >
    Kurilian
  </option>
  <option
    value="LaPerm"
  >
    LaPerm
  </option>
  <option
    value="Maine-Coon"
  >
    Maine Coon
  </option>
  <option
    value="Malayan"
  >
    Malayan
  </option>
  <option
    value="Manx"
  >
    Manx
  </option>
  <option
    value="Munchkin"
  >
    Munchkin
  </option>
  <option
    value="Nebelung"
  >
    Nebelung
  </option>
  <option
    value="Norwegian-Forest Cat"
  >
    Norwegian Forest Cat
  </option>
  <option
    value="Ocicat"
  >
    Ocicat
  </option>
  <option
    value="Oriental"
  >
    Oriental
  </option>
  <option
    value="Persian"
  >
    Persian
  </option>
  <option
    value="Pixie-bob"
  >
    Pixie-bob
  </option>
  <option
    value="Ragamuffin"
  >
    Ragamuffin
  </option>
  <option
    value="Ragdoll"
  >
    Ragdoll
  </option>
  <option
    value="Russian-Blue"
  >
    Russian Blue
  </option>
  <option
    value="Savannah"
  >
    Savannah
  </option>
  <option
    value="Scottish-Fold"
  >
    Scottish Fold
  </option>
  <option
    value="Selkirk-Rex"
  >
    Selkirk Rex
  </option>
  <option
    value="Siamese"
  >
    Siamese
  </option>
  <option
    value="Siberian"
  >
    Siberian
  </option>
  <option
    value="Singapura"
  >
    Singapura
  </option>
  <option
    value="Snowshoe"
  >
    Snowshoe
  </option>
</select>
`;

exports[`<Select /> Snaphot must match dogSelectRender 1`] = `
<select
  className="select"
>
  <option
    value="affenpinscher"
  >
    affenpinscher
  </option>
  <option
    value="african"
  >
    african
  </option>
  <option
    value="airedale"
  >
    airedale
  </option>
  <option
    value="akita"
  >
    akita
  </option>
  <option
    value="appenzeller"
  >
    appenzeller
  </option>
  <option
    value="shepherd-australian"
  >
    shepherd australian
  </option>
  <option
    value="basenji"
  >
    basenji
  </option>
  <option
    value="beagle"
  >
    beagle
  </option>
  <option
    value="bluetick"
  >
    bluetick
  </option>
  <option
    value="borzoi"
  >
    borzoi
  </option>
  <option
    value="bouvier"
  >
    bouvier
  </option>
  <option
    value="boxer"
  >
    boxer
  </option>
  <option
    value="brabancon"
  >
    brabancon
  </option>
  <option
    value="briard"
  >
    briard
  </option>
  <option
    value="norwegian-buhund"
  >
    norwegian buhund
  </option>
  <option
    value="boston-bulldog"
  >
    boston bulldog
  </option>
  <option
    value="english-bulldog"
  >
    english bulldog
  </option>
  <option
    value="french-bulldog"
  >
    french bulldog
  </option>
  <option
    value="staffordshire-bullterrier"
  >
    staffordshire bullterrier
  </option>
  <option
    value="cairn"
  >
    cairn
  </option>
  <option
    value="australian-cattledog"
  >
    australian cattledog
  </option>
  <option
    value="chihuahua"
  >
    chihuahua
  </option>
  <option
    value="chow"
  >
    chow
  </option>
  <option
    value="clumber"
  >
    clumber
  </option>
  <option
    value="cockapoo"
  >
    cockapoo
  </option>
  <option
    value="border-collie"
  >
    border collie
  </option>
  <option
    value="coonhound"
  >
    coonhound
  </option>
  <option
    value="cardigan-corgi"
  >
    cardigan corgi
  </option>
  <option
    value="cotondetulear"
  >
    cotondetulear
  </option>
  <option
    value="dachshund"
  >
    dachshund
  </option>
  <option
    value="dalmatian"
  >
    dalmatian
  </option>
  <option
    value="great-dane"
  >
    great dane
  </option>
  <option
    value="scottish-deerhound"
  >
    scottish deerhound
  </option>
  <option
    value="dhole"
  >
    dhole
  </option>
  <option
    value="dingo"
  >
    dingo
  </option>
  <option
    value="doberman"
  >
    doberman
  </option>
  <option
    value="norwegian-elkhound"
  >
    norwegian elkhound
  </option>
  <option
    value="entlebucher"
  >
    entlebucher
  </option>
  <option
    value="eskimo"
  >
    eskimo
  </option>
  <option
    value="bichon-frise"
  >
    bichon frise
  </option>
  <option
    value="germanshepherd"
  >
    germanshepherd
  </option>
  <option
    value="italian-greyhound"
  >
    italian greyhound
  </option>
  <option
    value="groenendael"
  >
    groenendael
  </option>
  <option
    value="havanese"
  >
    havanese
  </option>
  <option
    value="afghan-hound"
  >
    afghan hound
  </option>
  <option
    value="basset-hound"
  >
    basset hound
  </option>
  <option
    value="blood-hound"
  >
    blood hound
  </option>
  <option
    value="english hound"
  >
    english hound
  </option>
  <option
    value="ibizan-hound"
  >
    ibizan hound
  </option>
  <option
    value="plott-hound"
  >
    plott hound
  </option>
  <option
    value="walker-hound"
  >
    walker hound
  </option>
  <option
    value="husky"
  >
    husky
  </option>
  <option
    value="keeshond"
  >
    keeshond
  </option>
  <option
    value="kelpie"
  >
    kelpie
  </option>
  <option
    value="komondor"
  >
    komondor
  </option>
  <option
    value="kuvasz"
  >
    kuvasz
  </option>
  <option
    value="labrador"
  >
    labrador
  </option>
  <option
    value="leonberg"
  >
    leonberg
  </option>
  <option
    value="lhasa"
  >
    lhasa
  </option>
  <option
    value="malamute"
  >
    malamute
  </option>
  <option
    value="malinois"
  >
    malinois
  </option>
  <option
    value="maltese"
  >
    maltese
  </option>
  <option
    value="bull-mastiff"
  >
    bull mastiff
  </option>
  <option
    value="english-mastiff"
  >
    english mastiff
  </option>
  <option
    value="tibetan-mastiff"
  >
    tibetan mastiff
  </option>
  <option
    value="mexicanhairless"
  >
    mexicanhairless
  </option>
  <option
    value="mix"
  >
    mix
  </option>
  <option
    value="bernese-mountain"
  >
    bernese mountain
  </option>
  <option
    value="swiss-mountain"
  >
    swiss mountain
  </option>
  <option
    value="newfoundland"
  >
    newfoundland
  </option>
  <option
    value="otterhound"
  >
    otterhound
  </option>
  <option
    value="papillon"
  >
    papillon
  </option>
  <option
    value="pekinese"
  >
    pekinese
  </option>
  <option
    value="pembroke"
  >
    pembroke
  </option>
  <option
    value="miniature-pinscher"
  >
    miniature pinscher
  </option>
  <option
    value="pitbull"
  >
    pitbull
  </option>
  <option
    value="german-pointer"
  >
    german pointer
  </option>
  <option
    value="germanlonghair-pointer"
  >
    germanlonghair pointer
  </option>
  <option
    value="pomeranian"
  >
    pomeranian
  </option>
  <option
    value="miniature-poodle"
  >
    miniature poodle
  </option>
  <option
    value="standard-poodle"
  >
    standard poodle
  </option>
  <option
    value="toy-poodle"
  >
    toy poodle
  </option>
  <option
    value="pug"
  >
    pug
  </option>
  <option
    value="puggle"
  >
    puggle
  </option>
  <option
    value="pyrenees"
  >
    pyrenees
  </option>
  <option
    value="redbone"
  >
    redbone
  </option>
  <option
    value="chesapeake-retriever"
  >
    chesapeake retriever
  </option>
  <option
    value="curly-retriever"
  >
    curly retriever
  </option>
  <option
    value="flatcoated-retriever"
  >
    flatcoated retriever
  </option>
  <option
    value="golden-retriever"
  >
    golden retriever
  </option>
  <option
    value="rhodesian-ridgeback"
  >
    rhodesian ridgeback
  </option>
  <option
    value="rottweiler"
  >
    rottweiler
  </option>
  <option
    value="saluki"
  >
    saluki
  </option>
  <option
    value="samoyed"
  >
    samoyed
  </option>
  <option
    value="schipperke"
  >
    schipperke
  </option>
  <option
    value="giant-schnauzer"
  >
    giant schnauzer
  </option>
  <option
    value="miniature-schnauzer"
  >
    miniature schnauzer
  </option>
  <option
    value="english-setter"
  >
    english setter
  </option>
  <option
    value="gordon-setter"
  >
    gordon setter
  </option>
  <option
    value="irish-setter"
  >
    irish setter
  </option>
  <option
    value="english-sheepdog"
  >
    english sheepdog
  </option>
  <option
    value="shetland-sheepdog"
  >
    shetland sheepdog
  </option>
  <option
    value="shiba"
  >
    shiba
  </option>
  <option
    value="shihtzu"
  >
    shihtzu
  </option>
  <option
    value="blenheim-spaniel"
  >
    blenheim spaniel
  </option>
  <option
    value="brittany-spaniel"
  >
    brittany spaniel
  </option>
  <option
    value="cocker-spaniel"
  >
    cocker spaniel
  </option>
  <option
    value="irish-spaniel"
  >
    irish spaniel
  </option>
  <option
    value="japanese-spaniel"
  >
    japanese spaniel
  </option>
  <option
    value="sussex-spaniel"
  >
    sussex spaniel
  </option>
  <option
    value="welsh-spaniel"
  >
    welsh spaniel
  </option>
  <option
    value="english-springer"
  >
    english springer
  </option>
  <option
    value="stbernard"
  >
    stbernard
  </option>
  <option
    value="american-terrier"
  >
    american terrier
  </option>
  <option
    value="australian-terrier"
  >
    australian terrier
  </option>
  <option
    value="bedlington-terrier"
  >
    bedlington terrier
  </option>
  <option
    value="border-terrier"
  >
    border terrier
  </option>
  <option
    value="dandie-terrier"
  >
    dandie terrier
  </option>
  <option
    value="fox-terrier"
  >
    fox terrier
  </option>
  <option
    value="irish-terrier"
  >
    irish terrier
  </option>
  <option
    value="kerryblue-terrier"
  >
    kerryblue terrier
  </option>
  <option
    value="lakeland-terrier"
  >
    lakeland terrier
  </option>
  <option
    value="norfolk-terrier"
  >
    norfolk terrier
  </option>
  <option
    value="norwich-terrier"
  >
    norwich terrier
  </option>
  <option
    value="patterdale-terrier"
  >
    patterdale terrier
  </option>
  <option
    value="russell-terrier"
  >
    russell terrier
  </option>
  <option
    value="scottish-terrier"
  >
    scottish terrier
  </option>
  <option
    value="sealyham-terrier"
  >
    sealyham terrier
  </option>
  <option
    value="silky-terrier"
  >
    silky terrier
  </option>
  <option
    value="tibetan-terrier"
  >
    tibetan terrier
  </option>
  <option
    value="toy-terrier"
  >
    toy terrier
  </option>
  <option
    value="westhighland-terrier"
  >
    westhighland terrier
  </option>
  <option
    value="wheaten-terrier"
  >
    wheaten terrier
  </option>
  <option
    value="yorkshire-terrier"
  >
    yorkshire terrier
  </option>
  <option
    value="vizsla"
  >
    vizsla
  </option>
  <option
    value="spanish-waterdog"
  >
    spanish waterdog
  </option>
  <option
    value="weimaraner"
  >
    weimaraner
  </option>
  <option
    value="whippet"
  >
    whippet
  </option>
  <option
    value="irish-wolfhound"
  >
    irish wolfhound
  </option>
</select>
`;
